Storage/ThirdParty/SwiftData.swift

```diff
@@ -917,9 +906,8 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
         var status = sqlite3_close(db)
 
         if status != SQLITE_OK {
-            log.error("Got status \(status) while attempting to close.")
-            SentryIntegration.shared.sendWithStacktrace(message: "Got status \(status) while attempting to close.", tag: "SwiftData", severity: .error)
-            
+            Sentry.shared.sendWithStacktrace(message: "Got error status while attempting to close.", tag: SentryTag.swiftData, severity: .error, description: "error status: \(status)")
```

Comment:
Change to: `Sentry.shared.sendWithStacktrace(message: "Unable to close database.", tag: SentryTag.swiftData, severity: .error, description: "SQLite status: \(status)")`